stories/components/Conversation.js

```diff
@@ -5,16 +5,16 @@ import StoryItem from '../styleguide/StoryItem';
 
 const mockComments = [{
   id: 11,
-  body: 'Comment body...',
-  createdAt: 'today',
+  body: 'Here is a decent size comment that was created by someone who wanted to comment.',
+  createdAt: 'Less than a minute ago',
   person: {
     avatar: 'https://gathercontent-production-avatars.s3-us-west-2.amazonaws.com/208205_yHGd7vA5HRxsnMQpES4UzjJ7Yxgn6Bp54165gqksRXyDJhuOnW88H6djhLJeE2BZ.jpg',
```

Comment:
Shall we leave handling the lack of an image URL falling back to the user's initials for later? (more than happy to do this a later stage)